src/time_zone_info.cc

```diff
@@ -919,7 +919,7 @@ bool TimeZoneInfo::NextTransition(const time_point<seconds>& tp,
     ++begin;
   }
   std::int_fast64_t unix_time = ToUnixSeconds(tp);
-  const Transition target = { unix_time };
+  const Transition target = { unix_time, {}, {}, {} };
```

Comment:
We might as well use the same form as line 819 (or change that to this).

Reply:
Copied the line 819 here.